service.yml

```diff
@@ -0,0 +1,16 @@
+name: kafka-rest-images
+lang: unknown
+lang_version: unknown
+codeowners:
+  enable: true
+git:
+  enable: true
+semaphore:
+  enable: true
+  pipeline_type: cp-dockerfile
+  docker_repos: ['confluentinc/cp-kafka-rest',]
+  maven_phase: 'package'
+  maven_skip_deploy: true
+  build_arm: true
+  cve_scan: true
+  sign_image: true
```

Comment:
I believe this needs `nano_version: true`. https://github.com/confluentinc/confluent-security-plugins/pull/666